strings/nuclear_disk_quips.json

```diff
@@ -0,0 +1,188 @@
+{
+	"Clingy Timer Start Message":
+	[
+		"Get me back inside you dimwit!",
+		"Get me back inside you fucking idiot!",
+		"I am GOING to be upset.",
+		"I hate it out here!",
+		"I wanna go home to my bed!",
+		"I'm not built for this!",
+		"It was warm in there, wasn't it? Certainly warmer than whatever the fuck this is.",
+		"It's fucking horrific out here!",
+		"My programming wasn't enough to cope with this!",
+		"Oh my God, this is so fucking horrible!",
+		"This is a divisive matter... you wish to be outside, while I wish to remain inside.",
+		"You know how I have a tendency to disappear sometimes? Yeah.",
+		"What the fuck do you think you're doing? I'm still here, y'know."
+	],
+	"In Space":
+	[
+		"Didja pack your grappling hook?",
+		"Don't fall onto another plane of existence!",
+		"Don't go too far out. I don't want to be left behind.",
+		"Given enough time, I wonder if you can touch the ice moon...",
+		"I heard they might have space whales out here...",
+		"I kinda wish I was back in my comfy bed rather than be out here in this... vastness...",
+		"I'm picking up on all that delicious ionizing radiation.",
+		"Kinda hope you don't clumsy out and slip on literally nothing, dropping me into the void.",
+		"Lavaland seems so big out here...",
+		"Space is so big, and I'm so small...",
+		"There's a whole lotta stars out here...",
+		"They don't have many songs about space, do they? I can't think of one.",
+		"Watch out for carps!",
```

Comment:
do we use carp or carps? I forget

Reply:
Codebase search for "carps" reveals a whole lotta inconsistency.